portfolio/src/main/webapp/index.html

```diff
@@ -5,14 +5,188 @@
+      <p class="intro">Among my favorite extracurriculars you can find volleyball and swimming for sports, playing the piano and dancing for the arts, and speaking French for the languages.</p>
+      <p class="intro">I am very involved in my SHPE Chapter and Familia, where I currently serve as Internal Vicepresident, managing recruitment, retention and cohesion among its members. I also enjoy my Resident Advisor work on campus, where I assist Freshmen with their college transition.</p>
+      <p class="intro">Due to my origins, I have acquired a very global perspective, so I very much value individual aptitudes that add up to the collective interest. I regard myself as very resilient and committed, and look forward to opportunities like this program to put my technological preparation into practice. Hands on!</p>
+    </div>
+    <span class="span1">
+    <div class="project-showcase">
+        &nbsp;
+        &nbsp;
+        &nbsp;
+        &nbsp;
+        <ol>
+            <li>CONTACT INFO:</li>
+            <li><hr style="height:5px;background-color:#d9d5fc;color:#d9d5fc;border-width:0;width:150px;margin-left:0px"></li>
+            <li><a href="#https://www.linkedin.com/in/sofia-fernandez-chacin-242787175/">LinkedIn</a>  <a href="#https://github.com/sofifch">GitHub</a>  <a href="https://www.instagram.com/sofifch/">Instagram</a></li>
+        </ol>
+        &nbsp;
+        &nbsp;
+        <h2 id="projectshowcase">PROJECT SHOWCASE</h2>
+        <hr style="height:5px;background-color:#6495ED;color:#6495ED;border-width:0;width:400px">
+        <hr style="height:5px;background-color:#d9d5fc;color:#d9d5fc;border-width:0;width:1400px;margin-bottom:40px">
+        <div class="row">
+            <div class="column" style="width:400px">
+                <img src="images/tesis.jpg" style="width:320px;height:260px;margin-bottom:15px">
+                <hr style="height:5px;background-color:#6495ED;color:#6495ED;border-width:0;width:400px">
+                <p class="project-title">Manufacture of a Baby Incubator Prototype</p>
+                <p class="project-dsc">In a team of 3 (with my friends Andrea and Gabriela),  we performed a 2-year investigation to determine the essential functions of an incubator. 
+                    Then, we designed a prototype that implemented the vital functions using materials accessible for low-resource populations.
+                    After performing functionality validation tests, we donated several prototypes to foundations in need. </p>
+            </div>
+            <div class="column" style="width:400px">
+                <hr style="height:5px;background-color:#6495ED;color:#6495ED;border-width:0;width:400px">
+                <p class="project-title">UI Features of a Cash Flow Mobile App</p>
+                <p class="project-dsc">During my internship experience at Ernst & Young Consulting, I had the chance of helping a team that was participating in their internal “Shark-Tank” style competition.
```

Comment:
We have a coding style rule for maximum 80 character lines so I tend to have all my windows narrow and prefer to wrap things by hand. I don't have an opinion about HTML (so no need to change this) but I would probably ask you to wrap Java code.

Reply:
I agree that it was not ideal to have to side-scroll to see the complete line of code. I am going to be honest, I did not count the characters per line, but I did narrow it down by hand to keep it around 80-90 characters.

Reply:
Many editors can be configured to put a vertical line at a specified number of characters.